debian/control

```diff
@@ -7,6 +7,7 @@ Build-Depends:
   cargo,
   rustc (>=1.36.0),
   libgtk-3-dev,
+  libxml2-dev,
```

Comment:
This should be avoidable by using `default-features = false` with the gtk-extras dependency, if the svg feature there isn't needed.

Perhaps that feature shouldn't be default...

Reply:
Though it may not matter, really. While it adds a dev dependency, gtk probably pulls in libxml at runtime anyway.